modules/decision/search_pattern.py

```diff
@@ -0,0 +1,98 @@
+"""
+Initiates/continues a search pattern when the drone cannot find a landing pad.
+"""
+
+from .. import decision_command
+from .. import odometry_and_time
+from math import tan, pi, cos, sin, ceil
+
+class SearchPattern:
+    """
+    Attributes:
+        camera_fov (float): 
+            Camera's field of view, measured in degrees. Use the smallest measurement available.
+            Is essentially assumed to be a circle
```

Comment:
does the field of view refer to the the angle between the line to the furthest point the camera can see and straight down, or double that angle (I'm assuming it's double the angle)? You don't have to change anything here, just remember to put this in the documentation